react/tic-tac-toe/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import LoginForm from '../Login/components/LoginForm';
+
+class Login extends Component {
+  submit = (values) => {
+    this.props.checkCredentials(values);
+  };
+
+  render() {
+      return (
+        <LoginForm
+          onSubmit={this.submit}
+          isLoading={this.props.loading}
+          hasError={this.props.hasError ? true : false}
```

Comment:
I thiink hasError should work directly, you don't need to make the conditional
`hasError={this.props.hasError}`